PIL/Image.py

```diff
@@ -2142,6 +2237,7 @@ def __transformer(self, box, image, method, data,
         self.im.transform2(box, image.im, method, data, resample, fill)
```

Comment:
No, it doesn't appear to be used.  I'm going to say historical anomaly, as it's passed in and used in ImageTransform, but here in transform2 it's hardwired to 1.